types/options.go

```diff
@@ -457,8 +457,11 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) erro
 	if config.Storage.RootlessStoragePath != "" {
 		storeOptions.RootlessStoragePath = config.Storage.RootlessStoragePath
 	}
-	for _, s := range config.Storage.Options.AdditionalImageStores {
-		storeOptions.GraphDriverOptions = append(storeOptions.GraphDriverOptions, fmt.Sprintf("%s.imagestore=%s", config.Storage.Driver, s))
+	if len(config.Storage.Options.AdditionalImageStores) > 0 {
+		imageStore := config.Storage.Options.AdditionalImageStores[0]
+		additionalImageStores := strings.Join(config.Storage.Options.AdditionalImageStores, ",")
```

Comment:
I'm not opposed to this way of doing things, but I think it'd be conceptually cleaner to add a new API field that was an array/list.

Squashing an array to a string like this comma-separated raises questions around e.g. "what happens if the store path has a comma in it" - to be clear not something that really we'd expect to happen in this case.

But as a *general* case these things can happen, and ad-hoc "stringification" can become a problem.

There's also the flip side in that in theory something could have been reading the prior field and relying on it only containing (the top?) store, and suddenly seeing multiple paths there would break a prior consumer.

Reply:
this can be specified multiple times, e.g. `podman --storage-opt overlay.additionalimagestore=/tmp/1 --storage-opt overlay.additionalimagestore=/tmp/2 ...` works fine.

So instead of combining multiple additional image stores in a single option, we can have multiple `additionalimagestore`, one for each directory.

Said so, we don't currently support commas in the path, as we have an explicit `strings.Split(val, ",")` in `overlay.go`.